src/main/java/org/folio/inventory/dataimport/consumers/DataImportKafkaHandler.java

```diff
@@ -146,7 +146,8 @@ public Future<String> handle(KafkaConsumerRecord<String, String> record) {
       eventPayload.getContext().put(CHUNK_ID_HEADER, chunkId);
       eventPayload.getContext().put(USER_ID_HEADER, userId);
 
-      Context context = EventHandlingUtil.constructContext(eventPayload.getTenant(), eventPayload.getToken(), eventPayload.getOkapiUrl());
+      Context context = EventHandlingUtil.constructContext(eventPayload.getTenant(), eventPayload.getToken(), eventPayload.getOkapiUrl(),
```

Comment:
please pass userId during context creation in the OrdersPreloaderHelper as well
https://github.com/folio-org/mod-inventory/blob/cb2b9368c07872adc0c7d5d0048fd99d714481f6/src/main/java/org/folio/inventory/dataimport/handlers/matching/preloaders/OrdersPreloaderHelper.java#L54

Reply:
thank you Ruslan, nice catch!

Reply:
but does mod-orders require this header as well? i guess only mod-inventory-storage requires it?

Reply:
here in this class we need to pass userId just to avoid "invalid token" error during request that is performed in this class. Thank you Javokhir!